modules/apps/app-builder/app-builder-service/service.xml

```diff
@@ -49,6 +48,28 @@
 
 		<reference entity="User" package-path="com.liferay.portal" />
 	</entity>
+	<entity local-service="true" name="AppBuilderAppDeployment" remote-service="false">
+
+		<!-- PK fields -->
+
+		<column name="appBuilderAppDeploymentId" primary="true" type="long" />
+
+		<!-- Other fields -->
+
+		<column name="appBuilderAppId" type="long" />
+		<column name="deploymentType" type="String" />
+		<column name="settings" type="String" />
```

Comment:
is it a JSON array? better to make it a CLOB?

Reply:
done